contracts/Groups.sol

```diff
@@ -0,0 +1,80 @@
+pragma solidity ^0.4.24;
+
+
+contract Group {
+    function isMember(address _account) public view returns(bool);
+
+    modifier onlyMember() {
+        require(isMember(msg.sender), "Not a member of this group");
+
+        _;
+    }
+
+    function forward(address _to, bytes4 _selector, bytes _args) public payable onlyMember {
+        /// TODO: Figure out how to best forward the call
+    }
+}
+
+
+contract FixedGroup is Group {
+    mapping(address => bool) members;
+
+    constructor(address[] _members) public {
```

Comment:
Please comment the limitation of _members length

Reply:
Anyway to calculate the limitation?